src/FlaUI.WebDriver.UITests/ElementTests.cs

```diff
@@ -9,40 +10,70 @@ namespace FlaUI.WebDriver.UITests
     [TestFixture]
     public class ElementTests
     {
-        [Test]
-        public void GetText_Label_ReturnsRenderedText()
+        [TestCase("TextBox", "Test TextBox")]
+        [TestCase("PasswordBox", "●●●●●●●●●●")]
```

Comment:
This differs from WinAppDriver in that there a `????????` string is returned, however I feel that the existing behavior of returning `●` characters is less ambiguous so I left it.